web-components/src/[sandbox]/examples/draggable.ts

```diff
@@ -0,0 +1,288 @@
+import "@/components/draggable/Draggable";
+import "@/components/list/List";
+import "@/components/list/ListItem";
+import "@/components/checkbox/Checkbox";
+import "@/components/checkbox/CheckboxGroup";
+import "@/components/icon/Icon";
+import { comboBoxOptions } from "@/[sandbox]/sandbox.mock";
+import reset from "@/wc_scss/reset.scss";
+import { List } from "@/components/list/List";
+import { ListItem } from "@/components/list/ListItem";
+import { css, customElement, html, LitElement, query, queryAll } from "lit-element";
+import Sortable, { GroupOptions } from "sortablejs";
+
+@customElement("default-draggable-sandbox")
+export class DefaultDraggable extends LitElement {
+  @query("md-list") list!: List.ELEMENT;
+  @queryAll("md-list-item") listItem!: ListItem.ELEMENT[];
+
+  handleDragEnd(event: CustomEvent<{ srcEvent: Sortable.SortableEvent }>) {
+    const {
+      detail: {
+        srcEvent: { newIndex, newDraggableIndex }
+      }
+    } = event;
+
+    if (newIndex) {
+      this.list.activated = newIndex;
+      return false;
+    }
+
+    if (newDraggableIndex) {
+      this.list.activated = newDraggableIndex;
+      return false;
+    }
+  }
+
+  handleDragMove(event: CustomEvent<{ srcEvent: Sortable.MoveEvent }>) {
+    const {
+      detail: {
+        srcEvent: { dragged }
+      }
+    } = event;
+
+    const draggedIndex = [...this.listItem].findIndex(listItem => listItem.isSameNode(dragged));
+
+    if (draggedIndex !== -1) {
+      this.list.activated = draggedIndex;
+    }
+  }
+
+  static get styles() {
+    return [
+      reset,
+      css`
+      md-draggable {
+        width: 100%;
+      }
+      .sorting {
+        opacity: 0.4;
+        border-style: dashed;
+        border-color: #333;
+      }
+      .md-row {
+        display: flex;
+        flex-direction: row;
+        justify-content: space-between;
+        align-items: center;
+        align-content: center;
+        border: 1px solid #DEDEDE;
+        background-color: #fff;
+        width: 100%;
+        padding: 5px 10px;
+      }
+      .fourth {
+        display: inline-flex;
+        align-items: center;
+      }
+      md-list-item::part(list-item) {
+        padding: 0;
+        margin: 1px;
+        background: #fff;
+      }
```

Comment:
in Figma the rows are right up against one another, and the margin here is creating spaces between. `div.md-row` also adds some border too

Reply:
Component not finish yet. Will update styles.